parachain/src/chain_spec/testnet_interlay.rs

```diff
@@ -69,16 +69,10 @@ pub fn staging_mainnet_config(benchmarking: bool) -> InterlayChainSpec {
                 Some(get_account_id_from_string(
                     "5Ec37KSdjSbGKoQN4evLXrZskjc7jxXYrowPHEtH2MzRC7mv",
                 )),
-                crate::chain_spec::interlay::PARA_ID.into(),
+                PARA_ID.into(),
                 DEFAULT_BITCOIN_CONFIRMATIONS,
-            );
-
-            genesis.btc_relay.bitcoin_confirmations = DEFAULT_BITCOIN_CONFIRMATIONS;
-            genesis.btc_relay.parachain_confirmations =
```

Comment:
How come this is no longer needed? Iirc this was needed for the benchmarks to run correctly. Are you sure they still run?